### ISSUE TITLE
Mouse buttons

### ISSUE THREAD

**Issue by Doogie13** [kami-blue/client#1746]
**Is your feature request related to a problem? Please describe.**
Allow for binding mouse buttons (1 - 5) via binds
**Describe the solution you'd like**
;bind holesnap mouse3

**Describe alternatives you've considered**
idk

**Additional context**
same names as in source games

**Comment by l1ving** [comment]
>  same names as in source games

That would be lwjgl

**Comment by l1ving** [comment]
Note: see #1016 and #53. I'm leaving this as low priority because I'm not sure how to go about this, and it would require a rewrite of some modules. Also blocked by #1510, as that implements a new configuration system.

**Comment by Doogie13** [comment]
Fair enough

**Comment by Doogie13** [comment]
bump because edits (I know this isn't possible without a macro rewrite just throwing the idea out there)

**Comment by l1ving** [comment]
Same issue as before
